src/concrete/ml/sklearn/base.py

```diff
@@ -1993,23 +1994,7 @@ def scatter1d(x, v, indices):
 
         return numpy.expand_dims(topk_labels, axis=0)
 
-    # KNN works only for MONO in the latest concrete Python version
-    # FIXME: https://github.com/zama-ai/concrete-ml-internal/issues/3978
     def compile(self, *args, **kwargs) -> Circuit:
-        # If a configuration instance is given as a positional parameter, set the strategy to
-        # multi-parameter
-        if len(args) >= 2:
-            configuration = force_mono_parameter_in_configuration(args[1])
```

Comment:
You need to delete the import of `from ..common.utils import force_mono_parameter_in_configuration` from base.py
And from src/concrete/ml/common/utils.py as well: 
````
from concrete.fhe import ParameterSelectionStrategy
from concrete.fhe.compilation.configuration import Configuration
```
+
```
def force_mono_parameter_in_configuration(configuration: Optional[Configuration], **kwargs) ....
````